### ISSUE TITLE
BaseVoltage.ttl issues

### ISSUE THREAD

**Issue by VladimirAlexiev** [Haigutus/Energy-Reference-Data#21]
I've converted files to turtle and will start posting issues per file.
If you answer specific items (bullets), please use the item number!
```ttl
<http://energy.referencedata.eu/BaseVoltage/b0966c6c-9e5e-11e5-a575-c01885dc2c80>
        rdf:type                        cim:BaseVoltage , skos:Concept ;
        cim:BaseVoltage.nominalVoltage  "130" ;
        cim:IdentifiedObject.description
                "Base voltage 130 kV" ;
        cim:IdentifiedObject.name       "AC-130" ;
        dcterms:identifier              "urn:uuid:b0966c6c-9e5e-11e5-a575-c01885dc2c80" ;
        skos:definition                 "Base voltage 130 kV"@en ;
        skos:inScheme                   <http://energy.referencedata.eu/BaseVoltage> ;
        skos:prefLabel                  "AC-130" ;
        skos:topConceptOf               <http://energy.referencedata.eu/BaseVoltage> .

<http://energy.referencedata.eu/BaseVoltage/55c363d2-b10c-46dd-b851-9a014b68d031>
        rdf:type                        skos:Concept , cim:BaseVoltage ;
        cim:BaseVoltage.nominalVoltage  "35" ;
        cim:IdentifiedObject.description
                "Base voltage defined by ENTSO-E 35 kV" ;
        cim:IdentifiedObject.name       "AC-35" ;
        dcterms:identifier              "urn:uuid:55c363d2-b10c-46dd-b851-9a014b68d031" ;
        skos:definition                 "Base voltage defined by ENTSO-E 35 kV"@en ;
        skos:inScheme                   <http://energy.referencedata.eu/BaseVoltage> ;
        skos:prefLabel                  "AC-35" ;
        skos:topConceptOf               <http://energy.referencedata.eu/BaseVoltage> .
```
- 1. remove `urn:uuid:` from `identifier`: the identifier is just the GUID, the rest turns it into a URI.
  To use the URI you need to write it `<urn:uuid:55c363d2-b10c-46dd-b851-9a014b68d031>` but it's not needed since you use a URL at energy.referencedata.eu
- 2. switch all URLs to use `https:` not `http:`. All modern sites use https, and browsers issue ugly warnings for http sites
- 3. It really would be better to use meaningful URLs not GUIDs (but only if you can guarantee stability!) 
  - Eg in this file you use the http://energy.referencedata.eu/BaseVoltage/, so there's no need to use GUIDs to guarantee uniqueness.
  - Eg http://energy.referencedata.eu/BaseVoltage/AC-35 would be easier to understand, especially in references, eg:
     `<resource> cim:baseVoltage <BaseVoltage/AC-35>`
  - 3c. Eg for parties and resources, I argued that EIC should be used in the URL. But @Sveino and @griddigit said this can't be done: can you explain why
- 4. `topConceptOf` is a bit redundant: `inScheme` is mandatory, and `topConceptOf` is only used in hierarchical thesauri (which this is not)
- 5. unify `definition`: IMHO remove "defined by ENTSO-E" since it's not present in all entries.
- 6. must define the http://energy.referencedata.eu/BaseVoltage `ConceptScheme`
- 7. use `dct` rather than `dcterms`, because it's more commonly used
- 8. is this prefix needed, and why `at` not eg `eu-auth`, and where can I read about it?
   `@prefix at:      <http://publications.europa.eu/ontology/authority/> .`

**Comment by VladimirAlexiev** [comment]
@kristjan.vilgo see item 1

**Comment by Haigutus** [comment]
1. urn:uuid is used to mark the data type, UUID is a number not a string and without the prefix, one can't know that it is not just a string

https://www.w3.org/TR/vocab-dcat-3/#Property:resource_identifier
https://terms.tdwg.org/wiki/dcterms:identifier

![image](https://user-images.githubusercontent.com/11408965/191600416-a09f827d-22e1-4c64-84c9-80df6e5ba778.png)

**Comment by Haigutus** [comment]
7. in DCAT it is used with dcterms prefix (picture/link) above

**Comment by Haigutus** [comment]
linked to: https://github.com/Haigutus/Energy-Reference-Data/issues/15

**Comment by VladimirAlexiev** [comment]
1: you don't use a URN but a URL (`<http://energy.referencedata.eu/BaseVoltage/b0966c6c-9e5e-11e5-a575-c01885dc2c80>`). `<urn:uuid:>` doesn't just say "next comes a uuid" but says "this is a URN", so you need to render it as a node not literal.

7: 
- in http://prefix.cc/popular/all `dct` is number 21 while `dcterms` is number 21.
- in https://www.w3.org/ns/dcat.ttl, https://www.w3.org/ns/dcat2.ttl, https://www.w3.org/ns/dcat-external.ttl it's `dct`
- But you have a point too. That's a minor discrepancy between the DCAT html and ttl, maybe raise an issue?

**Comment by Haigutus** [comment]
2: https is not standard practice nor is it a different protocol, the underlining protocol is still http, https just refers to use of TLS and is automatically negotiated. I find the below comment reasonable and future proof approach, where url-s do not need to reflect the security layer used in the connection (what if we move to something else than TLS some time in future - far fetched example) 

[Sandro Hawke](https://www.w3.org/People/Sandro) says:
[May 20, 2016 at 6:58 pm](https://www.w3.org/blog/2016/05/https-and-the-semantic-weblinked-data/#comment-93683)
In short, keep writing “http:” and trust that the infrastructure will quietly switch over to TLS (https) whenever both client and server can handle it. Meanwhile, let’s try to get SemWeb software to be doing TLS+UIR+HSTS and be as secure as modern browsers.https://www.w3.org/blog/2016/05/https-and-the-semantic-weblinked-data/

**Comment by Haigutus** [comment]
3: We had long discussions on that one. The final decision to use UUID was based on risk that people start to parse the data from URI and really the linked data is geared towards IT systems, thus human readability of ID-s is less important.

**Comment by Haigutus** [comment]
4: topConceptOf was and is in use with the long term plan to align with EU Commission Data Publication office practices and this is needed for VockBench (Tool that Data Publication office is using) to show and display data in a correct way.
